docs/usage/control_plane_migration.md

```diff
@@ -10,6 +10,47 @@ Also, the involved Seeds need to have enabled BackupBuckets.
 
 `ShootState` is an API resource which stores non-reconstructible state and data required to completely recreate a `Shoot`'s control plane on a new `Seed`.  The `ShootState` resource is created on `Shoot` creation in its `Project` namespace and the required state/data is persisted during `Shoot` creation or reconciliation.
 
+## `shoots/binding` subresource
```

Comment:
Can you rather move this documentation to https://github.com/gardener/gardener/blob/master/docs/concepts/scheduler.md and refer to it from here?